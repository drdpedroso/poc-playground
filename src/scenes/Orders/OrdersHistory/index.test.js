import React from 'react'
import {OrdersHistory} from './index'
import renderer from 'react-test-renderer'

describe('OrdersHistory', () => {
  it('should render correctly', () => {
    const component = renderer.create(<OrdersHistory />)
    expect(component).toMatchSnapshot()
  })
})
