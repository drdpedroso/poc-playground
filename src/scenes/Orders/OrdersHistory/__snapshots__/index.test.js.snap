// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersHistory should render correctly 1`] = `
<div>
  leke3333
</div>
`;
